### PR TITLE
Support swiper in card launcher module.

### LINKED ISSUE #3: GPEA | Card Launcher Details
截圖 2020-03-19 上午9 44 09

Now this module only support exactly 3 cards. And it looks awful on mobile. 

## Change I: Support empty `Title`
截圖 2020-03-19 上午10 49 45

Although the `Title` section is not marked as required field, the module hide itself when there's no `Title` provides.

### Requirements
1. Please make the `Title` is NOT required. 
2. Make sure the layout is fine without the title. (The empty title should not occupy any space)

## Change II: Support dynamic add/remove cards.

add_remove_cards

### Requirements

1. Add two button on the bottom `Add more launcher`, `Remove the last laumcher`
2. When click on `Add more launcher`，add one more section with following fields:
    
    * **Launcher N Title***
    * Launcher N Paragraph 
    * Launcher N Optional image
    * Launcher N Button label
    * Launcher N Button link

Where the N equals to the index of this module. ex 1, 2, 3, 4, 5 ...

3. There's no limit of max number of launchers. User can add as many as they want. 
4. When click on `Remove the last launcher`, remove the last launcher. (Please do a little survey, can we delete the specific launcher instead the last one? ex. Add a remove button after each launcher. Or re-order the launchers.)
5. When there's on only launcher left, user click the `Remove the last launcher`. Clean all content of that launcher. 

## Change III: Change the layout on mobile version

Now the mobile version layout the three cards in vertical.
綠色和平東亞分部   Greenpeace 綠色和平   香港(1)

We want to change it into horizontal layout. (The same layout as `GPEA | Big carousel - manual selection Details`).

### Requirements

1. Be able to display more than 3 cards.
2. Change the mobile layout from vertical to horizontal.
3. Be able to slide left or right on both pc and mobile. 
4. In pc, it should have the next and prev button.

Target layout
horizontal_layout

### PR DESCRIPTION
**Pull request by wallat**
Fix #3 Card Launcher Detail

**Please also review this PR in repo [planet4-gpea-plugin-blocks](https://github.com/greenpeace/planet4-gpea-plugin-blocks/pull/6)**

Updates:

After:
截圖 2020-03-26 下午4 43 43

### Support PC/mobile swiper

> * In the original module, it doesn't support the swipe.
> * The module display cards vertically on the mobile version which  makes the page super long.
> Before:
> 綠色和平東亞分部   Greenpeace 綠色和平   香港(1)

After:

*PC version*
* Activate the swiper when there are more than or equal 4 cards.
UP_TEST_PAGE___Greenpeace

*Mobile version*
* Active swiper anyway
未命名-2

### CODE REVIEW

**Comment by eahk** [comment]
Good to go

File: frontend/js/components/swipers.js [hunk 1]
```diff
@@ -183,7 +183,7 @@ export default function(Swiper) {
         centeredSlides: true,
         simulateTouch: false,
         loop: false,
-        initialSlide: 1,
+        initialSlide: window.innerWidth<1023 ? 0:1,
```
Comment (eahk) [inline frontend/js/components/swipers.js]: Why is initialSlide set to 0 when innerWidth < 1023?

Comment (hi-upchen) [reply frontend/js/components/swipers.js]: On PC, it looks better if the initial slide is 1.
截圖 2020-03-26 下午5 37 52

On mobile, initial slide = 1 will hide the first card. That's why the initial slide should be zero on mobile.
FireShot Capture 012 - UP TEST PAGE - Greenpeace - www planet4 test

Comment (eahk) [reply frontend/js/components/swipers.js]: got it

**Pull request merged**